include/cantera/numerics/CVodesIntegrator.h

```diff
@@ -66,6 +65,26 @@ class CVodesIntegrator : public Integrator
     }
     virtual double sensitivity(size_t k, size_t p);
 
+    //! Get nonlinear solver stats from integrator
+    virtual AnyMap nonlinearSolverStats() const;
+
+    //! Get linear solver stats from integrator
+    //! @param stats a long int pointer with at least eight spaces
+    virtual AnyMap linearSolverStats() const;
+
+    //! Set the linear solver type. Default is DENSE+NOJAC.
+    /*!
+     * @param linSolverType    Type of the linear solver.
+     */
+    void setLinSolverType(int linSolverType) {
```

Comment:
@speth This could be a good time to deprecate the integer constants. I don't actually see the use of `JAC` in the `applyOptions` function but only `NOJAC` so I am not sure what the function of these arguments are. There isn't for example a `DENSE+JAC` option.